keps/sig-cli/3515-kubectl-explain-openapiv3/README.md

```diff
@@ -491,37 +456,23 @@ Defined using feature gate
 - Plaintext output roughly matches explain output
 - OpenAPIV3 (raw json) output implemented
 
-#### Alpha 2
-
-If we decide to move ahead with the `md` and `html` outputs, an Alpha 2 may
-be required.
-
-- `md` output implemented (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Section for each individual GVK
-  - All types hyperlink to specific section
--  basic `html` output  (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Page for each individual GVK.
-  - All types hyperlink to their specific page
-  - Searchable by name, description, field name.
-
 #### Beta
 
-- kube-openAPI v3 JSON deserialization is optimized to take less than 150ms on
```

Comment:
This is still mentioned as a requirement in the "OpenAPI serialization time" section. Was this improvement merged, or are we removing the requirement without it, and if so why?

Reply:
Part of the original design plan was to depend upon kube-openapi's `spec.Swagger` type. The type at the time had very slow unmarshalling. Since then the explain implementation design changed to deserialize into just `map[string]any` which does not have the same performance concerns.

Reply:
Should this be removed too then?

https://github.com/kubernetes/enhancements/blob/dea911f9a3f4035d766161e3938c63e2162046b8/keps/sig-cli/3515-kubectl-explain-openapiv3/README.md?plain=1#L318-L328

Reply:
yup! missed that. thank you!